README.md

```diff
@@ -153,15 +129,43 @@ organization to this list!
 - ["Shift-Left Security: The Path To PostgreSQL On Kubernetes" by Gabriele Bartolini](https://www.tfir.io/shift-left-security-the-path-to-postgresql-on-kubernetes/) (April 2021)
 - ["Local Persistent Volumes and PostgreSQL usage in Kubernetes" by Gabriele Bartolini](https://www.2ndquadrant.com/en/blog/local-persistent-volumes-and-postgresql-usage-in-kubernetes/) (June 2020)
 
-## Star History
+---
+
+<p align="center">
+We are a <a href="https://www.cncf.io/sandbox-projects/">Cloud Native Computing Foundation sandbox project</a>.
```

Comment:
probably we can put "Sandbox" since it's like a name? what do you think?

Reply:
I just used what the CNCF said.